IPython/html/static/base/js/dialog.js

```diff
@@ -32,7 +35,7 @@ define([
                 ).append(
                     $("<h4/>")
                         .addClass('modal-title')
-                        .text(options.title || "")
+                        .text(options.title || "No Title")
```

Comment:
I think actually having no title is better than giving the dialog a title called "No Title"